projects/js-packages/publicize-components/src/components/admin-page/toggles/social-image-generator-toggle/index.tsx

```diff
@@ -55,7 +60,7 @@ const SocialImageGeneratorToggle: React.FC< SocialImageGeneratorToggleProps > =
 				disabled={ isUpdating || ! isEnabled }
 				onClick={ open }
 			>
-				{ __( 'Change default template', 'jetpack-publicize-components' ) }
+				{ __( 'Change default Template and Image', 'jetpack-publicize-components' ) }
```

Comment:
Should we simply say "Configure" instead of "Change default Template and Image"?

Reply:
Fixed in 311cf0c

![CleanShot 2025-05-19 at 17 27 01 png](https://github.com/user-attachments/assets/b48bd584-13f7-4cfe-82ad-c2aa2eae6021)